packages/iframe-stamper/src/index.ts

```diff
@@ -233,6 +236,42 @@ export class IframeStamper {
     });
   }
 
+  /**
+   * Function to inject a new credential into the iframe
+   * The bundle should be encrypted to the iframe's initial public key
+   * Encryption should be performed with HPKE (RFC 9180).
+   * This is used during auth flows.
+   */
+  async injectAuthBundle(bundle: string): Promise<boolean> {
```

Comment:
except for the name / purpose, is there a difference between recovery and auth bundles? I'm thinking we should rename the event to "INJECT_CREDENTIAL_BUNDLE" and use it in both recovery and auth use cases?

Reply:
yep great idea! made the relevant changes here and in frames here: https://github.com/tkhq/frames/pull/19

still begs the question: got any suggestions for a good name?

Reply:
What do you think about `InjectCredentialBundle`? Are you happy with it or do you think we should try to find something better?

Reply:
I love `InjectCredentialBundle` ! Sorry, meant a name for the consolidated bundle -- and seems like we're good with `auth`